### DISCUSSION TITLE
"For you" errs with `<Twirp::Error code:internal msg:"error fetching feed items" meta:{}>`

### DISCUSSION THREAD

**Discussion by OfekShilon** [community/community#35216] [Other Feature Feedback, Questions, & Ideas] [answered]
_
![image](https://user-images.githubusercontent.com/73080/194116064-a1a326df-b5c1-4303-b5bb-12eb985c4295.png)

**Comment by troysjanda** [comment]
<Twirp::Error code:internal msg:"error fetching feed items" meta:{}> 

same error for me as well

**Comment by emmaviolet** [comment, marked as answer]
Thanks for reporting! I believe this should be addressed now.
